packages/protocol-kit/tests/e2e/core.test.ts

```diff
@@ -74,7 +158,43 @@ describe('Safe Info', () => {
   })
 
   describe('getAddress', async () => {
-    it('should return the Safe contract address', async () => {
+    itif(safeVersionDeployed < '1.3.0')(
+      'should fail to return the address of a Safe <v1.3.0 that is not deployed',
+      async () => {
+        const { predictedSafe, accounts, contractNetworks } = await setupTests()
+        const [account1] = accounts
+        const ethAdapter = await getEthAdapter(account1.signer)
+        const safeSdk = await Safe.create({
+          ethAdapter,
+          predictedSafe,
+          contractNetworks
+        })
+        const getSafeAaddress = safeSdk.getAddress()
+        chai
+          .expect(getSafeAaddress)
+          .to.be.rejectedWith(
+            'Account Abstraction functionality is not available for Safes with version lower than v1.3.0'
+          )
+      }
+    )
+
+    itif(safeVersionDeployed >= '1.3.0')(
+      'should return the address of a Safe >=v1.3.0 that is not deployed',
+      async () => {
+        const { predictedSafe, accounts, contractNetworks } = await setupTests()
+        const [account1] = accounts
+        const ethAdapter = await getEthAdapter(account1.signer)
+        const safeSdk = await Safe.create({
+          ethAdapter,
+          predictedSafe,
+          contractNetworks
+        })
+        const getSafeAaddress = safeSdk.getAddress()
+        chai.expect(await getSafeAaddress).to.be.eq(await safeSdk.getAddress())
```

Comment:
Here we test that the address returned by the same function is the same when calling it twice. Could it make sense to set explicitly the Safe address? I understand that the configuration should be constant on all test environments so that way we can ensure it returns what we expect